hack/test-cluster-class.sh

```diff
@@ -13,14 +13,27 @@ if [[ ! -f "./.envfile" ]]; then
 fi
 source ./.envfile
 
-clusterctl generate yaml --from templates/clusterclass-template.yaml > ./output/generated-clusterclass-template-replaced.yaml
-clusterctl generate cluster $CLUSTER_NAME --from templates/cluster-template-topology.yaml > ./output/generated-cluster-template-topology.yaml
-#
-#echo "validate"
+echo "Setup Cluster API with ClusterResourceSet and ClusterTopology"
+echo "export EXP_CLUSTER_RESOURCE_SET=\"true\""
+echo "export CLUSTER_TOPOLOGY=\"true\""
+echo "clusterclt init --infrastructure=ionos-cloud"
 
-echo "done"
-#clusterctl alpha topology plan -f ./templates/clusterclass-template-replaced.yaml -f ./templates/cluster-template-topology-replaced.yaml -o output/
+GENERATED_CLUSTER_CLASS_FILE="./output/generated-clusterclass-template.yaml"
+GENERATED_CLUSTER_FILE="./output/generated-cluster-template-topology-calico.yaml"
 
-# echo "apply"
-#kubectl apply -f ./templates/clusterclass-template-replaced.yaml
-#kubectl apply -f ./templates/cluster-template-topology-replaced.yaml
+clusterctl generate yaml --from ./templates/clusterclass-template.yaml > $GENERATED_CLUSTER_CLASS_FILE
+clusterctl generate cluster $CLUSTER_NAME -n $NAMESPACE --from ./templates/cluster-template-topology-calico.yaml > $GENERATED_CLUSTER_FILE
+
+echo "Validate generated resources"
+clusterctl alpha topology plan -f $GENERATED_CLUSTER_CLASS_FILE -f $GENERATED_CLUSTER_FILE -o output/
+
+echo "Apply cluster class resources"
+echo "kubectl apply -f $GENERATED_CLUSTER_CLASS_FILE"
+echo "Apply cluster which uses the cluster class resources"
+echo "kubectl apply -f $GENERATED_CLUSTER_FILE"
+
+echo "Installing calico cni"
+echo "make crs-calico"
+echo "kubectl apply -f ./templates/crs/cni/calico.yaml"
```

Comment:
You have created a variable with the root project directory. Please use it for specifiying paths for input and output files

Reply:
fixed